server/Login.class.php

```diff
@@ -294,13 +294,11 @@ public function register($email){
 				$mail_body .= "<br />You can reach us at: <a href='mailto:".$this->contact_email."'>".$this->contact_email."</a>";
 			}
 
-			## Build Email Headers
-			$headers = "From: ".$this->noreply_email."\r\n";
-			$headers .= 'MIME-Version: 1.0' . "\r\n";
-			$headers .= 'Content-type: text/html; charset=iso-8859-1' . "\r\n";
```

Comment:
Loving the code clean-up. It's pretty dumb to be copy-pasting this stuff around.